### PR TITLE
Add TaDReP

### PR DESCRIPTION
**Pull request by oschwengers**
This PR add a recipe for a new tool called [TaDReP](https://github.com/oschwengers/tadrep): targeted detection & reconstruction of plasmids.

### CODE REVIEW

File: recipes/tadrep/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+package:
+  name: tadrep
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/oschwengers/tadrep/archive/v{{ version }}.tar.gz
+  sha256: '5dcba6aa04e4e87d985779dd6c7968a20550ebb1304defc59f66bba698239079'
+
+build:
+  noarch: python
```
Comment (bgruening) [inline recipes/tadrep/meta.yaml:12]: please use noarch: generic

It does contain non python dependencies

Comment (oschwengers) [reply recipes/tadrep/meta.yaml:12]: Are runtime deps also taken into account? I thought this only relates to build deps? TaDReP itself is pure Python, therefore I thought `noarch: python` would be the right tag.

**Comment by oschwengers** [comment]
@BiocondaBot please add label

File: recipes/tadrep/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+test:
+  import:
+    - Bio.SeqIO
+    - pyrodigal
+  commands:
+    - tadrep --help
+
+about:
+  home: https://github.com/oschwengers/tadrep
+  license: GPLv3
```
Comment (bgruening) [inline recipes/tadrep/meta.yaml:41]: can you use here please the SPDX identifier?

Comment (oschwengers) [reply recipes/tadrep/meta.yaml]: done

**Review by bgruening — APPROVED** [review]
Thanks @oschwengers

**Comment by oschwengers** [comment]
Thanks a lot @bgruening !

**Comment by bgruening** [comment]
Your welcome, keep rocking!

**Pull request merged**